diff --git a/containers/ci-front/Dockerfile b/containers/ci-front/Dockerfile
index 5bc8ef983..373eb0f46 100644
--- ./containers/ci-front/Dockerfile
+++ ./containers/ci-front/Dockerfile
@@ -1,6 +1,6 @@
 FROM node:13-alpine
 
-RUN apk add --no-cache \
+RUN apk add  \
     git \
     libsass
 
diff --git a/containers/live-server-compiler/Dockerfile b/containers/live-server-compiler/Dockerfile
index 1d74d7f7d..217994630 100644
--- ./containers/live-server-compiler/Dockerfile
+++ ./containers/live-server-compiler/Dockerfile
@@ -1,6 +1,7 @@
 FROM node:13-alpine
 
+RUN apk update && apk upgrade && \
+        apk add --no-cache bash git openssh
+
 COPY monitor.sh /monitor.sh
 COPY build.sh /build.sh
-
-ENTRYPOINT ["sh", "/monitor.sh"]
diff --git a/containers/live-server-compiler/monitor.sh b/containers/live-server-compiler/monitor.sh
old mode 100644
new mode 100755
index 7a41406d8..23f204c4b
--- ./containers/live-server-compiler/monitor.sh
+++ ./containers/live-server-compiler/monitor.sh
@@ -1,5 +1,7 @@
 #!/usr/bin/env sh
 set -e
 
-cd /var/www/Minds/front
-npx nodemon --delay 3 --watch server.ts --watch dist/en/ --ext js,css,jpg,png,svg,mp4,webp,webm  --exec "/usr/bin/env sh" /build.sh
+cd /var/www/Minds/front/
+
+npm i && \
+    NODE_OPTIONS=--max_old_space_size=5096 npm run build:dev --watch=false
